jax/linear_util.py

```diff
@@ -63,10 +63,13 @@ def trans1(static_arg, *dynamic_args, **kwargs):
 """
 
 import threading
+from functools import partial
 from typing import Any, Tuple, Callable
 import weakref
 
+from . import core
 from ._src.util import curry
+from .tree_util import tree_map
```

Comment:
Seems like you don't use this?

Reply:
Actually it is used: on a line that says `@partial(partial, tree_map)`, which I believe you and I wrote together like 6 months ago, probably while joking about how funny it was :)